src/activities/facebook/device_location.rs

```diff
@@ -13,24 +13,41 @@ pub struct CellPhoneCarrier {
 #[allow(non_snake_case)]
 #[derive(Deserialize, Debug)]
 pub struct DeviceLocation {
-  pub phone_number_location_v2: Vec<CellPhoneCarrier>
+  pub phone_number_location_v2: Option<Vec<CellPhoneCarrier>>,
```

Comment:
Could you update the commit message because changing the type is kind of an important change to others.

Reply:
Changing commit messages can be a bit hard, but this could be fixed when smashing and merging the pull request. That time the commit message can be written to give a summary of the changes.

Reply:
Well, I updated the commit message. Tell me if it is ok, or if I need to be more explicit.